app/src/main/java/com/rob729/quiethours/Adapter/ProfileListAdapter.kt

```diff
@@ -136,4 +136,13 @@ class ProfileListAdapter(
             }
             .show()
     }
+    fun deleteAll() {
+        val size: Int = profiles.size
+        if (size > 0) {
```

Comment:
unnecessary if check

Reply:
I will remove this

Reply:
@rob729 I have already updated the changes and pushed the code